smack-integration-test/src/main/java/org/jivesoftware/smackx/muc/MultiUserChatIntegrationTest.java

```diff
@@ -131,7 +131,7 @@ public void processMessage(Message message) {
             }
         });
 
-        createMuc(mucAsSeenByOne, "one-" + randomString);
+        createMuc(mucAsSeenByOne, Resourcepart.from("one-" + randomString));
```

Comment:
Since this is a re-occuring pattern, I think it would improve the readability of the tests if there was a 

```
public createMuc(MultiUserChat muc, String nick) {
   createMuc(muc, Resourcepart.from("one-" + randomString)
}
```
wrapper. This would also decrease the modified LoC count.